packages/contracts/test/StabilityPoolTest.js

```diff
@@ -1101,262 +822,14 @@ contract('StabilityPool', async accounts => {
       await th.fastForwardTime(timeValues.SECONDS_IN_ONE_HOUR, web3.currentProvider)
 
       // B tops up
-      await stabilityPool.provideToSP(dec(100, 18), frontEnd_1, { from: B })
+      await stabilityPool.provideToSP(dec(100, 18), { from: B })
 
       const G_After = await stabilityPool.epochToScaleToG(0, 0)
 
       // Expect G has increased from the LQTY reward event triggered by B's topup
       assert.isTrue(G_After.gt(G_Before))
     })
 
-    it("provideToSP(), topup from different front end: doesn't change the front end tag", async () => {
-      await openTrove({ extraLUSDAmount: toBN(dec(10000, 18)), ICR: toBN(dec(10, 18)), extraParams: { from: whale } })
-
-      // whale transfer to troves D and E
-      await lusdToken.transfer(D, dec(100, 18), { from: whale })
-      await lusdToken.transfer(E, dec(200, 18), { from: whale })
-
-      // A, B, C open troves 
-      await openTrove({ extraLUSDAmount: toBN(dec(100, 18)), ICR: toBN(dec(2, 18)), extraParams: { from: A } })
-      await openTrove({ extraLUSDAmount: toBN(dec(200, 18)), ICR: toBN(dec(2, 18)), extraParams: { from: B } })
-      await openTrove({ extraLUSDAmount: toBN(dec(300, 18)), ICR: toBN(dec(2, 18)), extraParams: { from: C } })
-
-
-      // A, B, C, D, E provide to SP
-      await stabilityPool.provideToSP(dec(10, 18), frontEnd_1, { from: A })
-      await stabilityPool.provideToSP(dec(20, 18), frontEnd_2, { from: B })
-      await stabilityPool.provideToSP(dec(30, 18), ZERO_ADDRESS, { from: C })
-      await stabilityPool.provideToSP(dec(40, 18), frontEnd_1, { from: D })
-      await stabilityPool.provideToSP(dec(50, 18), ZERO_ADDRESS, { from: E })
-
-      await th.fastForwardTime(timeValues.SECONDS_IN_ONE_HOUR, web3.currentProvider)
-
-      // A, B, C, D, E top up, from different front ends
-      await stabilityPool.provideToSP(dec(10, 18), frontEnd_2, { from: A })
-      await stabilityPool.provideToSP(dec(20, 18), frontEnd_1, { from: B })
-      await stabilityPool.provideToSP(dec(15, 18), frontEnd_3, { from: C })
-      await stabilityPool.provideToSP(dec(20, 18), frontEnd_2, { from: D })
-      await stabilityPool.provideToSP(dec(30, 18), frontEnd_3, { from: E })
-
-      const frontEndTag_A = (await stabilityPool.deposits(A))[1]
-      const frontEndTag_B = (await stabilityPool.deposits(B))[1]
-      const frontEndTag_C = (await stabilityPool.deposits(C))[1]
-      const frontEndTag_D = (await stabilityPool.deposits(D))[1]
-      const frontEndTag_E = (await stabilityPool.deposits(E))[1]
-
-      // Check deposits are still tagged with their original front end
-      assert.equal(frontEndTag_A, frontEnd_1)
-      assert.equal(frontEndTag_B, frontEnd_2)
-      assert.equal(frontEndTag_C, ZERO_ADDRESS)
-      assert.equal(frontEndTag_D, frontEnd_1)
-      assert.equal(frontEndTag_E, ZERO_ADDRESS)
-    })
-
-    it("provideToSP(), topup: depositor receives LQTY rewards", async () => {
```

Comment:
do we still want this test?

Reply:
this is removed as the LQTY events have been removed from stability pool changes